src/hooks/useClipboard.test.ts

```diff
@@ -0,0 +1,23 @@
+import {act, renderHook} from '@testing-library/react-hooks';
+import {useClipboard} from '.';
+
+Object.defineProperty(navigator, 'clipboard', {
+  value: {
+    writeText: jest.fn(),
+  },
+});
+
+describe('useClipboard', () => {
+  jest.spyOn(navigator.clipboard, 'writeText');
```

Comment:
When and how to mock functions would be a cool FE Chapter topic. This file is a pretty solid example.

Reply:
Yeah, it's interesting stuff. I know just enough about it to get things done (usually), but probably not enough to explain it 😬